source/manual/data-gov-uk-deployment.html.md

```diff
@@ -50,10 +50,19 @@ deployed.
 The deployment for both apps can be triggered automatically via GitHub flow, or
 manually via command line tools.
 
+#### Staging
 The deployment to staging is triggered when a PR gets merged into master. You
-can check the Travis logs of the `master` build to see progress. The deployment
-to production is triggered when a [new release] with an appropriate version
-number is created in GitHub. This behaviour is defined in [Publish's travis.yml]
+can check the [Travis logs of the `master` build](https://travis-ci.org/alphagov/datagovuk_find) to see progress.
+
+You can check the deployment on [staging.data.gov.uk](https://staging.data.gov.uk/)
+
+#### Production
+The deployment to production is triggered when a [new release] with an appropriate version
+number is created in GitHub.
+
+To create a new release, provide a new version number, release title and description.
```

Comment:
Worth adding an example version number here, as in the past people have added a version number without the leading `v` and it hasn't deployed